src/pages/index.tsx

```diff
@@ -52,8 +54,39 @@ function HomePage() {
         </Modal>
       )}
       {isAddOpen && (
-        <Modal onClose={onAddClose}>
-          <h1>Hello world!</h1>
+        <Modal className={styles["modal-wrapper"]} onClose={onAddClose}>
```

Comment:
@ajsaule I'd highly recommend to create separate components for these modals. Something like 'TaskView' & 'TaskEdit'